src/main/java/core/basesyntax/controller/BetHandler.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.Dao;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class BetHandler implements ConsoleHandler {
```

Comment:
let's have only one handler that will read info about both entities from the console
example of user input `200 0.6 Masha 20`

Reply:
ok, if the task requires a connection between User and Bet, it would be nice to mention it in the task description.  Btw on today's lecture, Dmytro said they (User and Bet) don't necessarily have to be connected.  In my case, I thought that User isn't a gambler and the User doesn't make bets (the user has no connection to bet) it just two different examples for practicing DAO. This is why I made two handlers that can be used separately and independently.
Please, let me know if I still need to change the handlers or not

Reply:
I've made the required changes